### ISSUE TITLE
Create random tests in TimeSheetArithmeticTest class

### ISSUE THREAD

**Issue by gstuer** [kit-sdq/TimeSheetGenerator#82]
_Just for documentation purposes_

If it says that we need those random tests we should make an issue for them and solve this problem :)

_Originally posted by @gstuer in https://github.com/kit-sdq/TimeSheetGenerator/pull/79_

**Comment by larsk21** [comment]
Good idea. Would it be helpful to wait until we merged #74?

**Comment by gstuer** [comment]
Yes you're completely right! This issue should be blocked until #74 is merged.
